LayoutTests/fast/dom/StyleSheet/css-builder-converter-initial-letter-crash.html

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
```

Comment:
This is kinda a weird place for this test. Should probably be in fast/css/css-typed-om/ (where you can find similar crashes like style-converter-convert-shape-value-type-confusion.html)